openmc/mgxs/library.py

```diff
@@ -717,3 +721,267 @@ def load_from_file(filename='mgxs', directory='mgxs'):
 
         # Load and return pickled Library object
         return pickle.load(open(full_filename, 'rb'))
+
+    def write_mg_library(self, xs_type='macro', domain_names=None, xs_ids=None,
+                         filename='mg_cross_sections', directory='./',
+                         return_names=False):
+        """Creates a cross-section data library file for the Multi-Group
+        mode of OpenMC.
+
+        Parameters
+        ----------
+        xs_type: {'macro', 'micro'}
+            Provide the macro or micro cross section in units of cm^-1 or
+            barns. Defaults to 'macro'. If the Library object is not tallied by
+            nuclide this will be set to 'macro' regardless.
+        domain_names : Iterable of str
+            List of names to apply to the xsdata entries in the
+            resultant mgxs data file. Defaults to 'set1', 'set2', ...
+        xs_ids : str or Iterable of str
+            Cross section set identifier (i.e., '71c') for all
+            data sets (if only str) or for each individual one
+            (if iterable of str). Defaults to '1m'.
+        filename : str
+            Filename for the pickle file. Defaults to 'mg_cross_sections'.
```

Comment:
Never mind, i like mgxs.xml better!  Switching now.  

I've updated the docs and defaults to match.

And it is true that ".xml" is appended.